Gems/Atom/Tools/AtomToolsFramework/Code/Source/Graph/GraphUtil.cpp

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AtomToolsFramework/Graph/GraphUtil.h>
+
+namespace AtomToolsFramework
```

Comment:
What is this file for?

Reply:
Placeholder for other shared graph utility functions as I extract them from material canvas